metabase/metabase
# fix wrong visualization setting key format when combining cards on a dashboard

### Description

Fixes invalid viz settings key creating for combo chart series which led to empty charts when combining two cards on a dashboard.

### How to verify

- Create:
  - Single metric card
  - Multiple metrics card
  - Breakout card
- Create a dashboard and combine them
- Ensure the chart is rendered correctly

### Demo

Dynamic viz:
<img width="1047" alt="Screenshot 2024-01-16 at 8 40 22 PM" src="https://github.com/metabase/metabase/assets/14301985/ddf92d7f-6c6a-432b-ba7a-f525c35ed4be">

Static viz:
<img width="677" alt="Screenshot 2024-01-16 at 8 40 29 PM" src="https://github.com/metabase/metabase/assets/14301985/c332cf82-5fe8-4a73-a999-caed85ae8797">

### Checklist
